doc/nrf/device_guides/nrf52/fota_update.rst

```diff
@@ -111,7 +111,7 @@ See how to build the :ref:`peripheral_lbs` sample with MCUboot in direct-xip mod
    If the application is built without this option, it will fail to boot.
    It must, however, be disabled when building update images.
 
-Both the :kconfig:option:`SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP` and :kconfig:option:`SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP_WITH_REVERT` Kconfig options automatically build application update images for both slots.
+Both the ``SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP`` and ``SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP_WITH_REVERT`` Kconfig options automatically build application update images for both slots.
 To read about the files that are built when the option is enabled, refer to the :ref:`app_build_mcuboot_output` page.
 
 .. fota_upgrades_over_ble_mcuboot_direct_xip_nrfcdm_note_start
```

Comment:
Should we also mention `nRF54L` in line 198? I think it's also supported.

Reply:
Line 195: Should we suggest using HCI IPC sample or NCS IPC radio application? @dchat-nordic, what do you think? (IIRC you were involved in moving towards IPC radio in the NCS)